frontend/src/metabase-lib/lib/metadata/Database.ts

```diff
@@ -101,28 +151,24 @@ export default class Database extends Base {
   }
 
   // QUESTIONS
-  newQuestion() {
-    return this.question()
+  newQuestion(): Question {
+    return this.question({})
       .setDefaultQuery()
       .setDefaultDisplay();
   }
 
-  question(
-    query = {
-      "source-table": null,
```

Comment:
Based on the types in `StructuredQuery`, `"source-table"` can't be `null`. An empty object for the `query` seems to work as a replacement.